starter-code/src/viking.js

```diff
@@ -1,11 +1,140 @@
 // Soldier
-function Soldier() {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = 300;
+        this.strength = 150;
```

Comment:
La idea del constructor es que this.health sea igual a health que es el parametro que se pasa para crear el objeto. De esta forma podemos crear varios objetos parecidos con algunos valores diferentes.